api/GPUOutOfMemoryError.json

```diff
@@ -14,9 +14,13 @@
           },
           "edge": "mirror",
           "firefox": {
+            "version_added": "preview",
+            "partial_implementation": true,
+            "notes": "Currently supported on Linux and Windows only. See <a href='https://bugzil.la/1616739'>bug 1616739</a>."
+          },
+          "firefox_android": {
             "version_added": false
           },
-          "firefox_android": "mirror",
           "ie": {
```

Comment:
Note, the constructor subfeature is commented out in idl - which should mean "not implemented" - right?

Reply:
Sounds like it to me. You could try testing it using the code snippet available at https://developer.mozilla.org/en-US/docs/Web/API/GPUOutOfMemoryError#examples

Reply:
Will do, tomorrow. Thanks for the advice.